include/mem.h

```diff
@@ -19,6 +19,7 @@
 #ifndef DOSBOX_MEM_H
 #define DOSBOX_MEM_H
 
+#include <cstring>
```

Comment:
Suggestion: let's rewrite lines 22-25 to:
```
#include "dosbox.h"

#include <cstring>

#include "types.h"
```

The reason being: `dosbox.h` is kind of "main" header, most code receive C_* defines via "config.h" included in there - in case of this file, WORDS_BIGENDIAN and C_UNALIGNED_MEMORY.

"types.h" is alread being included via "dosbox.h", but maybe it's better to make it explicit.

Reply:
Thanks; added!